ledger/src/blockstore_db.rs

```diff
@@ -2024,6 +2030,18 @@ fn get_db_options(access_type: &AccessType) -> Options {
     options
 }
 
+/// The default number of threads to use for rocksdb compaction in the rocksdb
+/// low priority threadpool
+pub fn default_num_compaction_threads() -> NonZeroUsize {
+    NonZeroUsize::new(num_cpus::get()).expect("thread count is non-zero")
```

Comment:
I thought this was currently getting set to num cpus * 3/4?

Reply:
It does not. I provided some commentary for why this is the case in the PR problem description. See the "Additional Context for setting threadpool size directly" section which includes some links to the underlying rocksdb code